components/tcp_transport/transport_ws.c

```diff
@@ -97,7 +97,6 @@ static int ws_connect(esp_transport_handle_t t, const char *host, int port, int
                          "Host: %s:%d\r\n"
                          "Upgrade: websocket\r\n"
                          "Sec-WebSocket-Version: 13\r\n"
-                         "Sec-WebSocket-Protocol: mqtt\r\n"
```

Comment:
Please either leave the line or sync with mqtt for parametric configuration (as mentioned in generic comment)